### DISCUSSION TITLE
Can't find any documentation on "duration" option/function

### DISCUSSION THREAD

**Discussion by camillo777** [tweenjs/tween.js#686] [Q&A]
Is "duration" parameter expressed as? seconds, frames, milliseconds?
When using an array of "to" values, how Tweenjs handle the duration parameter?
Thank you

**Comment by mk965** [comment]
1. Is "duration" parameter expressed as? seconds, frames, milliseconds?
The duration parameter in Tween.js is expressed in milliseconds. For example, a duration of 1000 means the animation will take 1 second to complete.

2. When using an array of "to" values, how Tweenjs handle the duration parameter?
    When you provide an array of to values, Tween.js will treat it as a sequence of tweens. Each segment will have its own duration and interpolation will occur sequentially.
    ```js
    new TWEEN.Tween(object)
      .to({ x: [100, 200, 300] }, 6000) // Total duration is 6000 milliseconds
      .start();
    ```
    In this example, the total duration is 6000 milliseconds (6 seconds). Tween.js will divide the total duration by the number of segments (3 in this case), so each segment will have a duration of 2000 milliseconds (2 seconds).
